packages/iris-grid/src/IrisGridTreeTableModel.ts

```diff
@@ -182,6 +191,15 @@ class IrisGridTreeTableModel extends IrisGridTableModelTemplate<
       )
   );
 
+  getCachedGroupedColumnSet = memoize(
+    (groupedColumns: Column[]) =>
```

Comment:
Can be explicit about this being a `readonly` array:
```suggestion
    (groupedColumns: readonly Column[]) =>
```